apps/client/src/components/Location/LocationInput.js

```diff
@@ -12,16 +12,19 @@ import Nav from "../Nav";
 import LocationFinder from "./LocationFinder";
 
 const LocationInput = ({ topic, setActiveState, resetChecker }) => {
+  const history = useHistory();
   const { trackEvent } = useMatomo();
+  const { clearErrors, errors, register, unregister, handleSubmit } = useForm();
   const sessionContext = useContext(SessionContext);
   const checkerContext = useContext(CheckerContext);
-  const history = useHistory();
+
   const { slug, text, sttrFile } = topic;
   const sessionAddress = sessionContext[slug]?.address || {};
+  const useSTTR = !!sttrFile;
```

Comment:
All react hooks functions are prefixed with `use` so maybe we could name it hasSTTR or something.

Reply:
Good point, will change.